lib/document.js

```diff
@@ -119,6 +119,8 @@ class Document extends Readable {
 
     // start to work the _pending queue
     this._next()
+
+    this._rootFragment = new Fragment(this, this);
```

Comment:
I am wondering whether we could just call `Fragment.prototype._init.call(this, this, this)` and don't need the `_rootFragment` property at all (but wow, `this, this, this` looks ugly 😱)